pinc/Project.inc

```diff
@@ -243,32 +243,6 @@ class Project
         return $charsuites;
     }
 
-    public function get_valid_codepoints()
```

Comment:
You need to update the other instances where this is called -- `pinc/DPage.inc` and `pinc/bad_bytes.inc`

Reply:
Yes, this occured to me later also.

Reply:
New commit should avoid needing to do this.